### PR TITLE
implement chapter 4 intro page

### LINKED ISSUE #627: Implement CH4 Intro
Pretty straightforward, we should create intro pages for Chapter 4

[Link to Figma](https://www.figma.com/file/LqjK3Tpvd9KJ4buFArCJBQ/Saving-Satoshi?type=design&node-id=1535-19614&mode=design&t=bUUrHbu3heWV0c7c-0)

### PR DESCRIPTION
**Pull request by sabakhilji**
Closes #627 
Preview:
Screenshot (114)

### CODE REVIEW

**Comment by 0tuedon** [comment]
@sabakhilji  can you rebase from dev. i think a clean commit would be preferred

**Comment by benalleng** [comment]
You will need to add the intro to the chapter index to be able to test and access this intro page

**Comment by sabakhilji** [comment]
> @sabakhilji can you rebase from dev. i think a clean commit would be preferred

Hey @Otuedon I am facing some problems in rebasing. There are some merge conflicts. Can you please help me with that?

**Comment by ecurrencyhodler** [comment]
@sabakhilji can you update the title of this PR to something more descriptive? And you can add this in the description of the PR: closes #647

**Comment by ecurrencyhodler** [comment]
Is this ready for testing?

**Comment by sabakhilji** [comment]
ready for testing @ecurrencyhodler

**Comment by ecurrencyhodler** [comment]
Hmmm when I go to https://saving-satoshi-b1gv4ct0v-savingsatoshi.vercel.app/en/chapters/chapter-4/intro-1?dev=true I see a missing page:
Screenshot 2023-09-28 at 10 51 48 AM

File: content/chapters/chapter-4/index.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ export const metadata = {
   slug: 'chapter-4',
   title: 'chapter_four.title',
   image: '/assets/images/chapter-4-cover.jpg',
-  intros: [],
+  intros: ['intro'],
```
Comment (benalleng) [inline content/chapters/chapter-4/index.tsx:20]: We typically use "intro-1" even if there is only 1 page in the intro category.

**Pull request closed without merging**